packages/apps/src/element/id/AssetIdenticon.ts

```diff
@@ -34,9 +34,14 @@ export class AssetIdenticon extends LitElement {
   iconTemplate(id: string, icon: string) {
     const asset = this.assets.get(id);
 
-    const warning = !isExternalAssetWhitelisted(asset)
-      ? i18n.t('asset.warning.external')
-      : undefined;
+    const warning =
```

Comment:
I'm thinking maybe better approach would be to build warning/danger icon here and pass it as slotted to AssetId component. That way we don't have to change API of assetId with every change and we can keep it clean for extension by slotting. So maybe to create named slot badge instead ?

Reply:
Let's do this in next iteration we need this out.